src/test/java/com/cappella/csv/CsvClientTests.java

```diff
@@ -0,0 +1,165 @@
+package com.cappella.csv;
+
+import java.beans.Transient;
+import java.io.InputStream;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Locale;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.commons.csv.CSVRecord;
+import com.cappella.model.Problems;
+import com.cappella.model.TaskData;
+import com.cappella.model.SubTask;
+
+class CsvClientTests {
+
+        final Logger LOGGER = LoggerFactory.getLogger(getClass());
+
+        @Test
+        void testNoHeadersCsvFile() {
+                // the simplest way to malform the header is to add a trailing comma
+                ClassLoader classloader = this.getClass().getClassLoader();
+                InputStream is = classloader.getResourceAsStream("noHeaders.csv");
+                Problems problems = new Problems();
```

Comment:
If you choose to leave Problems as an external object that gets passed in to the client instances, make it a field in the test instead of instantiating it in each class.

Reply:
fixed in updated PR